model-optimizer/mo/utils/versions_checker.py

```diff
@@ -160,11 +215,25 @@ def check_requirements(framework=None):
     """
     if framework is None:
         framework_suffix = ""
+    elif framework == "tf":
+        try:
+            exec("import tensorflow")
+            tf_version = sys.modules["tensorflow"].__version__
+            if tf_version < "2.0.0":
```

Comment:
It is not correct to compare strings. Because "10.0.0" is less than "2.0.0" :)
Please, convert these numbers to integer and perform comparison of lists.

Reply:
Fixed.